pkgs/os-specific/linux/open-iscsi/default.nix

```diff
@@ -1,22 +1,23 @@
-{ stdenv, fetchurl, nukeReferences }:
-let
-  pname = "open-iscsi-2.0-873";
-in stdenv.mkDerivation {
-  name = pname;
+{ stdenv, lib, fetchFromGitHub, nukeReferences, automake, autoconf, libtool, gettext, utillinux, openisns, openssl, kmod }:
```

Comment:
Typically, `lib` is accessed via `stdenv.lib` so having it as a parameter is unnecessary.